server/models/admin.sql

```diff
@@ -0,0 +1,105 @@
+DELIMITER $$
+
+/*
+ zRecomputeEntityMap
+
+ Abolishes and recomputes the entity_map from the base tables in the system.  This is
+ useful in case of database corruption in which references get out of sync.
+*/
+CREATE PROCEDURE zRecomputeEntityMap()
```

Comment:
Inspired by Care2X and whatever accounting suite it embedded into them.  Denotes to developers that these will never be exposed to the application